pkg/agent/plugin/keymanager/disk/disk_test.go

```diff
@@ -61,12 +62,14 @@ func TestDisk_FetchPrivateKey(t *testing.T) {
 
 func TestDisk_Configure(t *testing.T) {
 	plugin := New()
+	tmpdir := os.TempDir()
 	cReq := &spi.ConfigureRequest{
-		Configuration: "directory = \"foo/bar\"",
+		Configuration: fmt.Sprintf("directory = \"%s\"", tmpdir),
 	}
 	_, e := plugin.Configure(ctx, cReq)
 	assert.NoError(t, e)
-	assert.Equal(t, "foo/bar", plugin.dir)
+	assert.Equal(t, tmpdir, plugin.dir)
+	assert.DirExists(t, tmpdir)
```

Comment:
since `os.TempDir()` already exists, this doesn't really ensure that the plugin creates the directory. I'd suggest something like:

```
tempDir, err := ioutil.TempDir("", "km-disk-test")
require.NoError(t, err)
defer os.RemoveAll(tmpdir)

keysDir := filepath.Join(tmpDir, "keys")

.... configure with keysDir ....
.... assert keysDir exists ....
```

Reply:
Oh, I made a mistake...
Thank you for your suggesting. I fixed with https://github.com/spiffe/spire/pull/899/commits/50964a0181e1cfaae3ee23e5f70a3e70e897e9c2 .